src/typescript/Scala.tmLanguage.ts

```diff
@@ -572,7 +572,7 @@ export const scalaTmLanguage: TmLanguage = {
       ]
     },
     'char-literal': {
-      end: "'",
+      end: "'|([^'\\s]*\\s(?=[^'(//)]))|$",
```

Comment:
I am not completely convinced that `([^'\\s]*\\s(?=[^'(//)]))` is an improvement. I opened https://github.com/scala/vscode-scala-syntax/pull/61 in stead a a first step.